development/nautobot_config.py

```diff
@@ -117,26 +117,30 @@
 CACHEOPS_REDIS = parse_redis_connection(redis_database=1)
 
 # Enable installed plugins. Add the name of each plugin to the list.
-PLUGINS = ["nautobot_chatops", "nautobot_capacity_metrics"]
+PLUGINS = [
+    "nautobot_chatops",
+    # "nautobot_chatops.tests.bootstrap",
+    "nautobot_capacity_metrics",
+]
 
 # Plugins configuration settings. These settings are used by various plugins that the user may have installed.
 # Each key in the dictionary is the name of an installed plugin and its value is a dictionary of settings.
 PLUGINS_CONFIG = {
     "nautobot_chatops": {
-        "enable_slack": True,
```

Comment:
These should change to be environment variables, and default to false. Then we won't need to comment them out.

Reply:
Fixed